examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -38,22 +38,21 @@ struct FrontendNode {
 }
 
 #[no_mangle]
-pub extern "C" fn frontend_oak_main(handle: u64) -> i32 {
-    std::panic::catch_unwind(|| main(handle))
-        .unwrap_or(Err(oak::OakStatus::ERR_INTERNAL))
-        .err()
-        .unwrap_or(oak::OakStatus::OK)
-        .value()
+pub extern "C" fn frontend_oak_main(handle: u64) {
+    let _ = std::panic::catch_unwind(|| {
+        oak::set_panic_hook();
+        main(handle)
+    });
 }
 
-pub fn main(handle: u64) -> Result<(), oak::OakStatus> {
+pub fn main(handle: u64) {
     let node = FrontendNode::new();
-    oak::grpc::event_loop(
+    let _ = oak::grpc::event_loop(
```

Comment:
Handle this result value too (by logging or unwrapping).

Reply:
Done.